types/posthtml.d.ts

```diff
@@ -1,24 +1,23 @@
-export interface NodeAttributes {
+type MaybeArray<T> = T | T[]
+
+export interface PostHTMLNodeAttributes {
```

Comment:
💄 You can use namespace `PostHTML` instead of manually writing the prefix.

```ts
export namespace PostHTML {
  export type NodeAttributes = Record<string, string>;
}
```

Reply:
💄 Also you can use internal `Record` generic instead of index signature.